examples/ffd/ffd_cyberon.cmake

```diff
@@ -0,0 +1,196 @@
+target_include_directories(${TARGET_NAME} PUBLIC ${APP_INCLUDES} ${RTOS_CONF_INCLUDES})
+target_compile_definitions(${TARGET_NAME} PUBLIC ${APP_COMPILE_DEFINITIONS} THIS_XCORE_TILE=0)
+target_compile_options(${TARGET_NAME} PRIVATE ${APP_COMPILER_FLAGS})
+target_link_libraries(${TARGET_NAME} PUBLIC ${APP_COMMON_LINK_LIBRARIES})
+target_link_options(${TARGET_NAME} PRIVATE ${APP_LINK_OPTIONS})
+unset(TARGET_NAME)
+
+set(TARGET_NAME tile1_example_ffd_cyberon)
+add_executable(${TARGET_NAME} EXCLUDE_FROM_ALL)
+target_sources(${TARGET_NAME} PUBLIC ${APP_SOURCES})
+target_include_directories(${TARGET_NAME} PUBLIC ${APP_INCLUDES} ${RTOS_CONF_INCLUDES})
+target_compile_definitions(${TARGET_NAME} PUBLIC ${APP_COMPILE_DEFINITIONS} THIS_XCORE_TILE=1)
+target_compile_options(${TARGET_NAME} PRIVATE ${APP_COMPILER_FLAGS})
+target_link_libraries(${TARGET_NAME} PUBLIC ${APP_COMMON_LINK_LIBRARIES})
+target_link_options(${TARGET_NAME} PRIVATE ${APP_LINK_OPTIONS} )
+unset(TARGET_NAME)
+
+#**********************
+# Merge binaries
+#**********************
+merge_binaries(example_ffd_cyberon tile0_example_ffd_cyberon tile1_example_ffd_cyberon 1)
+
+#**********************
+# Create run and debug targets
+#**********************
+create_run_target(example_ffd_cyberon)
+create_debug_target(example_ffd_cyberon)
+
+#**********************
+# Create data partition support targets
+#**********************
+set(TARGET_NAME example_ffd_cyberon)
+set(DATA_PARTITION_FILE ${TARGET_NAME}_data_partition.bin)
+set(MODEL_FILE ${TARGET_NAME}_model.bin)
+set(FATFS_FILE ${TARGET_NAME}_fat.fs)
+set(FLASH_CAL_FILE ${LIB_QSPI_FAST_READ_ROOT_PATH}/lib_qspi_fast_read/calibration_pattern_nibble_swap.bin)
+
+add_custom_target(${MODEL_FILE} ALL
+    COMMAND ${CMAKE_COMMAND} -E copy ${CYBERON_COMMAND_NET_FILE} ${MODEL_FILE}
+    COMMENT
+        "Copy Cyberon NET file"
+    VERBATIM
+)
+
+create_filesystem_target(
+    #[[ Target ]]                   ${TARGET_NAME}
+    #[[ Input Directory ]]          ${CMAKE_CURRENT_LIST_DIR}/filesystem_support/${MODEL_LANGUAGE}
+    #[[ Image Size ]]               ${FILESYSTEM_SIZE_BYTES}
+)
+
+add_custom_command(
+    OUTPUT ${DATA_PARTITION_FILE}
+    COMMAND ${CMAKE_COMMAND} -E rm -f ${DATA_PARTITION_FILE}
+    COMMAND datapartition_mkimage -v -b 1
+    -i ${FLASH_CAL_FILE}:${CALIBRATION_PATTERN_DATA_PARTITION_OFFSET} ${FATFS_FILE}:${FILESYSTEM_DATA_PARTITION_OFFSET} ${MODEL_FILE}:${MODEL_DATA_PARTITION_OFFSET}
+    -o ${DATA_PARTITION_FILE}
+    DEPENDS
+        ${MODEL_FILE}
+        make_fs_${TARGET_NAME}
+        ${FLASH_CAL_FILE}
+    COMMENT
+        "Create data partition"
+    VERBATIM
+)
+
+set(DATA_PARTITION_FILE_LIST
+    ${DATA_PARTITION_FILE}
+    ${MODEL_FILE}
+    ${FATFS_FILE}
+    ${FLASH_CAL_FILE}
+)
+
+set(DATA_PARTITION_DEPENDS_LIST
+    ${DATA_PARTITION_FILE}
+    ${MODEL_FILE}
+    make_fs_${TARGET_NAME}
+)
+
+# The list of files to copy and the dependency list for populating
+# the data partition folder are identical.
```

Comment:
And again. If not needed maybe delete or add comment to explain

Reply:
actually it is not commented out, they are bracket comments: https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-comment